relay/sources/relayd/src/data/runlog.rs

```diff
@@ -142,12 +148,21 @@ impl TryFrom<(RunInfo, Vec<RawReport>)> for RunLog {
             .ok_or(Error::InconsistentRunlog)?
             .start_datetime;
 
-        let config_id = reports
+        // Try to extract a configId from start or end control reports
+        // If not present, use 0 as default to allow inserting the runlog
+        let config_id = match reports
             .iter()
-            .find(|r| r.event_type == "control" && r.component == "end")
-            .ok_or(Error::MissingEndRun)?
-            .key_value
-            .clone();
+            .find(|r| r.event_type == "control" && (r.component == "start" || r.component == "end"))
+        {
+            Some(r) => r.key_value.clone(),
+            None => {
+                warn!(
+                    "Missing start/end control reports in runlog, using {} as config id",
+                    DEFAULT_CONFIG_ID
+                );
+                DEFAULT_CONFIG_ID.to_string()
```

Comment:
i think it should be null. if the value doesn't exists, we cannot create one (it would confuse with having a reseted node)